core/src/variant.rs

```diff
@@ -745,14 +745,15 @@ impl_to_variant_for_int!(i8);
 impl_to_variant_for_int!(i16);
 impl_to_variant_for_int!(i32);
 impl_to_variant_for_int!(i64);
+impl_to_variant_for_int!(isize);
 
 macro_rules! godot_uint_impl {
     ($ty:ty) => (
         impl ToVariant for $ty {
             fn to_variant(&self) -> Variant {
                 unsafe {
                     let mut ret = sys::godot_variant::default();
-                    (get_api().godot_variant_new_uint)(&mut ret, u64::from(*self));
+                    (get_api().godot_variant_new_uint)(&mut ret, *self as u64);
```

Comment:
I'd prefer the old `u64::from()` version, as it's guaranteed to be lossless, while `as` is not!

So for `usize` (the only case where the bound `From<usize>` for u64 would fail should be made explicit maybe? The macro here is not that big, so I *personally* would like an explicit implementation for this.

Others might think differently, just a suggestion :)

Reply:
OK, I will try that.